.github/workflows/mvn_docker_image.yml

```diff
@@ -57,6 +57,26 @@ on:
                 default: ubuntu-latest
                 type: string
                 description: Set to 'self-hosted' if needed.
+            self_hosted_cache_endpoint:
+                required: false
+                default: 'truenas.local.lan'
+                type: string
+                description: required if runs_on == 'self-hosted'
```

Comment:
required in this context is a bit misleading, since you do not need to assign this input (as there is a default value).
```suggestion
                description: only applicable if runs_on == 'self-hosted'
```

Reply:
Ah, copy-paste from secrets :)